exercises/series/series.exs

```diff
@@ -0,0 +1,11 @@
+defmodule StringSeries do
+  @doc """
+  Given a string `s` and a positive integer `size`, return all substrings
+  of that size. If `size` is greater than the length of `s`, or less than 1,
+  return an empty list.
+  """
+  @spec slices(s :: String.t(), size :: integer) :: any
```

Comment:
For our type spec, it looks like the return type of this function could be more specific since our tests are always returning a list. Might as well be specific here, right?